internals-js/src/__tests__/subgraphValidation.test.ts

```diff
@@ -878,6 +878,30 @@ describe('@core/@link handling', () => {
     // Just making sure this don't error out.
     buildAndValidate(doc);
   });
+
+  it('allows defining a repeatable directive as non-repeatable but validates usages', () => {
```

Comment:
Reviewer notes: this test is unrelated to the issue and it wasn't meant to be here, but I just wrote it earlier (to prove to myself that the behaviour of the test does work as the test shows) and didn't noticed it was there. But seeing it now, I'm thinking: "it's a nice test to have and is it really worth opening a separate PR for this?". Happy to remove though if it bothers someone.